app/scripts/models/reliers/sync.js

```diff
@@ -35,19 +45,7 @@ define(function (require, exports, module) {
       var self = this;
       return Relier.prototype.fetch.call(self)
         .then(function () {
-          self.importSearchParam('context');
-
-          try {
-            self.importBooleanSearchParam('customizeSync');
-          } catch (e) {
-            // ignore it for now.
```

Comment:
startup error handling was completed long ago, `customizeSync` can now be validated.